src/main/java/com/nike/cerberus/client/auth/aws/InstanceRoleVaultCredentialsProvider.java

```diff
@@ -99,25 +107,112 @@ protected void authenticate() {
      * Parses and returns the AWS account ID from the instance profile ARN.
      *
      * @return AWS account ID
+     * @deprecated no longer used, will be removed
      */
     protected String lookupAccountId() {
```

Comment:
This method appears to no longer be used.

- [ ] delete method, if actually no longer used.

Reply:
I left it at deprecated in the very off chance someone extended this class.  We'll need a new major version of the client soon enough.  We can delete it then.

Reply:
I must have missed the deprecated annotation, sounds good.